File: src/main/java/com/googongill/aditory/service/dto/link/LinkInfo.java
package com.googongill.aditory.service.dto.link;

import lombok.Builder;
import lombok.Getter;

import java.time.LocalDateTime;

@Getter
@Builder
public class LinkInfo {
    private Long linkId;
    private String title;
    private String summary;
    private String url;
    private Boolean linkState;
    private LocalDateTime createdAt;
    private LocalDateTime lastModifiedAt;
}

File: src/main/java/com/googongill/aditory/service/dto/link/LinkInfo.kt
package com.googongill.aditory.service.dto.link

import java.time.LocalDateTime

data class LinkInfo(
    val linkId: Long?,
    val title: String,
    val summary: String?,
    val url: String,
    val linkState: Boolean,
    val createdAt: LocalDateTime?,
    val lastModifiedAt: LocalDateTime?
)